tla/etcdraft2.tla

```diff
@@ -0,0 +1,1035 @@
+--------------------------------- MODULE etcdraft2 ---------------------------------
```

Comment:
Why is there a second raft spec?

Reply:
I added it by mistake. Remove it in the new commit.